### PR TITLE
release v0.4.0

### PR DESCRIPTION
**Pull request by q384566678**
Signed-off-by: Zhou Hao <zhouhao@cn.fujitsu.com>

### CODE REVIEW

**Comment by hqhq** [comment]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,55 @@
...
+* filepath/clean: Add Windows support (#539).
+* validate: Add validation when host-specific is set (#495).
+* runtimetest: Add validation of cgroups (#93).
+* generate: Generator solaris application container configuration (#532).
+* generate: Add interface to remove mounts. (#544).
+* validation/linux_cgroups_*: Generate TAP output (and outside-validation cleanup) (#542).
+* generate: Windows-specific container configuration generate (#528).
+* runtimetest: Add validateSeccomp (#514).
+* validation: Add mount validation (#547).
+* *: Transition from tap Diagnostic(...) to YAML(...) (#533).
```
Comment (vbatts) [inline CHANGELOG.md]: this `*:` is causing italics for the doc, down to ...

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,55 @@
...
+* runtimetest: Fix error return (#505).
+* runtimetest: Move validateRlimits to defaultValidations (#506).
+* runtimetest: Make validateRlimits silent on Windows (#509).
+* runtimetest: Raise ConfigInRootBundleDir for missing config.json (#508).
+* generate: Change process-tty to process-terminal (#517).
+* generate: Fixed seccompSet (#513).
+* runtimetest: Remove debug info (#518).
+* generate: Fix error return (#520).
+* validate: Fix nil deference (#522).
+* generate: Fix DropProcessCapability* (#519).
```
Comment (vbatts) [inline CHANGELOG.md]: ... here where it ends the italics. No big deal, but may look funny.

Comment (q384566678) [reply CHANGELOG.md]: updated, PTAL.

**Comment by q384566678** [comment]
The vote has been [passed.](https://groups.google.com/a/opencontainers.org/forum/#!topic/dev/Y-0iY9We8Fw) @hqhq @liangchenye PTAL

**Comment by liangchenye** [comment]
LGTM

**Comment by hqhq** [comment]
LGTM

**Pull request merged**